taichi/transforms/inlining.cpp

```diff
@@ -0,0 +1,96 @@
+#include "taichi/transforms/inlining.h"
+#include "taichi/ir/analysis.h"
+#include "taichi/ir/ir.h"
+#include "taichi/ir/statements.h"
+#include "taichi/ir/transforms.h"
+#include "taichi/ir/visitors.h"
+
+namespace taichi {
+namespace lang {
+
+// Inline all functions.
+class Inlining : public BasicStmtVisitor {
+ public:
+  using BasicStmtVisitor::visit;
+  DelayedIRModifier modifier;
```

Comment:
Sorry we haven't enforced this before, but let's keep the fields private as much as we can.